modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -1070,7 +1070,7 @@ CV_ALWAYS_INLINE void absdiffc_short_store_c3(ushort* out_ptr, const v_int32& c1
                                               const v_int32& c4, const v_int32& c5,
                                               const v_int32& c6)
 {
-    constexpr int nlanes = static_cast<int>(v_uint16::nlanes);
+    constexpr int nlanes = static_cast<int>(v_uint16::max_nlanes);
```

Comment:
why `max_nlanes` is used here, not `nlanes`?

Reply:
Yes should be `nlanes` here.